omod/src/main/java/org/openmrs/module/owa/web/controller/OwaRestController.java

```diff
@@ -218,4 +218,10 @@
 		}
 		return appList;
 	}
+
+	@RequestMapping(value = "/rest/owa/allowModuleWebUpload", method = RequestMethod.GET)
+	@ResponseBody
+	public boolean allowWebAdmin(HttpServletRequest request, HttpServletResponse response) {
+		 return ModuleUtil.allowAdmin();
```

Comment:
Not sure if this API end point needs specific privileges to access it, but @justmesam can you elaborate on this

Reply:
what privileges I don't understand.